.github/ISSUE_TEMPLATE/feature-proposal--developer-.md

```diff
@@ -43,24 +44,22 @@ assignees: ''
 > Include dynamic models (Activity Diagram, State Chart Diagram, Communication Diagram) here to outline the dynamic nature of the PROBLEM 
 
 
----
-## System Design 
+## System Architecture 
 
 ### Subsystem Decomposition
 > Show the involved subsystems and their interfaces. Make sure to describe the APIs that you add/change in detail. Model the DTOs you intend to (re)use or change! 
 
 ### Persistent Data Management
 > Describe the Database changes you intend to make.
-> Outline new config options you will add.
-> Describe all other data persistency mechanisms you may use. 
+> Outline new configuration options you plan to introduce
+> Describe all other data persistence mechanisms you may use.
 
 ### Access Control / Security Aspects 
 > Describe the access control considerations for your feature
 
 ### Other Design Decisions
-> Potential candidates to discuss here: Websockets, Test strategy 
+> Potential topics to discuss here include: WebSockets, testing strategies.
 
----
-## UI / UX 
+## UI/UX Design
 > Describe the user flow (references to dynamic model). 
-> Screenshots of the final UI mockup
+> Screenshots of the final UI mockup
```

Comment:
If I am not mistaken we want to use Figma for the Mockups?
I think this would be a good place to mention it at this place.

```suggestion
> Link to the [Figma UI mockup](https://www.figma.com/files/team/1238833580691926997/Artemis)
> Screenshots of the final UI mockup
```

Reply:
We've opted for screenshots over dynamic links for several reasons:

- Links to specific tools like Figma may require an account, which not everyone may have.
- We aim to provide all pertinent information directly within the issue itself to avoid necessitating additional clicks or potential registrations.
- Screenshots capture a stable state of for example the UI mockups that can be reliably referenced, whereas content behind a link may change, potentially making any feedback based on an earlier version outdated.

That being said, I’m open to including a link as an additional resource alongside the screenshots, but I would prefer to leave the decision to add such links to the discretion of the feature proposal creator. I still adjusted the template to also include the option to supplement the screenshots with a link.

Reply:
Yes, I agree with your points - I would rather have thought of it as additional information. The screenshots are definitely more important 👍